src/specklepy/objects/GIS/geometry.py

```diff
@@ -102,3 +102,48 @@ def __init__(
         self.displayValue = displayValue
         self.units = units or "m"
 
+class GisTopography(GisRasterElement, speckle_type="Objects.GIS.GisTopography", detachable={"displayValue"}):
+    """GIS Raster element"""
+
+    def __init__(
```

Comment:
Our classes do not follow the classic python class __init__ call for instance initailization, pls check the signature of the other Base objects